dashboard/app/views/levels/editors/_droplet.html.haml

```diff
@@ -21,13 +21,17 @@
   = boolean_check_box f, :lock_zero_param_functions
 
 .field
-  = f.label :text_mode_at_start, 'Text mode (showing code) at start'
-  = f.radio_button :text_mode_at_start, 'false'
-  = f.label :text_mode_at_start, 'Start with user preferred setting', :value => 'false'
-  = f.radio_button :text_mode_at_start, 'blocks'
-  = f.label :text_mode_at_start, 'Always start with blocks', :value => 'blocks'
-  = f.radio_button :text_mode_at_start, 'true'
-  = f.label :text_mode_at_start, 'Always start with text', :value => 'true'
+  = f.label :text_mode_at_start, 'Code View When Level Starts'
+  %p What mode the code is presented as when the level loads
+  .alignment-box
+    = f.radio_button :text_mode_at_start, 'false', :checked => true
```

Comment:
just double-checking: this just means the form will _default_ to this value, but will upon future loads be set to whatever it was changed to, right?

Reply:
Yes